code/modules/mob/living/carbon/xenomorph/abilities/defender/defender_powers.dm

```diff
@@ -1,9 +1,10 @@
 /datum/action/xeno_action/onclick/toggle_crest/use_ability(atom/target)
 	var/mob/living/carbon/Xenomorph/xeno = owner
+	var/datum/behavior_delegate/defender_base/defender_delegate = xeno.behavior_delegate
```

Comment:
Something feels off - I don't think it makes much sense to refactor the defender specific variables into a delegate, only to hard code that delegate to the defender. It's nice to pull vars out of the defender that don't need to be there but this makes me think it's an exercise in refactoring for the sake of it